ehr2vec/double_robust/control_code_handler.py

```
@@ -0,0 +1,67 @@
+    3. Inserting the control_code and associated data at the appropriate position
+    """
+    # Find event closest to index date
+    closest_event_idx = _get_closest_event_idx(index_date, patient_data)
+
+    # Determine insert position based on comparison with closest event
+    insert_idx = closest_event_idx
+    if index_date > patient_data["abspos"][closest_event_idx]:
+        insert_idx += 1
+
+    # Insert control code and associated data at determined position
+    for key in patient_data.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)